app/containers/Forms/CertificationSignature.tsx

```diff
@@ -89,7 +94,7 @@ export const CertificationSignature: React.FunctionComponent<Props> = ({
             />
           )}
         </Col>
-        <Col md={{span: 3, offset: 2}}>
+        <Col md={{offset: 3, span: 3}}>
```

Comment:
The offset changed here. Is this on purpose?

Reply:
Nope, I think I set my span for the rendered Toast message at 3, so I set the offset as the same. May as well change it back to what it was.